resources/todomvc/architecture-examples/vue/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "vue",
+  "version": "1.0.0",
+  "description": "TodoMVC app written with Vue.",
+  "private": true,
+  "engines": {
+      "node": ">=18.13.0",
+      "npm": ">=8.19.3"
+  },
+  "scripts": {
+    "serve": "vue-cli-service serve",
```

Comment:
In the other components (I looked at react / react-redux / preact), we used `start` for the development server and `serve` for serving the result of the build. `backbone` uses `dev` which is more explicit.
On the long run it would be good to be consistent, but since `serve` hasn't been used yet, how about using either `start` or `dev` here?

Reply:
yes, I am getting confused too. Ember-cli creates the scripts, but I'll rename to "dev"